esmvalcore/experimental/config/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""ESMValTool config module."""
+
+from ._config_object import CFG
+
+__all__ = [
+    'CFG',
+]
```

Comment:
```suggestion
    'Config',
    'Session',
]
It would be nice to include these here
```

Reply:
Disagreed, they are not meant to be called directly by users.